01week/javascripting/introduction.js

```diff
@@ -1 +1,78 @@
-console.log('hello');
+'use strict';
+
+// Write a JavaScript program to display the current day and time.
+
+function currentDateTime(){
+  console.log(new Date());
```

Comment:
The logic is right, however the function call is a little off. Your function should return the new date, and you should call the function in a console.log to display. 
`function currentDateTime(){
    return new Date();
  }
  console.log(currentDateTime());
`